httpclient/src/main/kotlin/tbdex/sdk/httpclient/TbdexHttpClient.kt

```diff
@@ -84,20 +87,41 @@ object TbdexHttpClient {
     val pfiServiceEndpoint = getPfiServiceEndpoint(pfiDid)
     val url = "$pfiServiceEndpoint/exchanges/$exchangeId/$kind"
```

Comment:
If `message` is a `Rfq`, this will send to the wrong endpoint. We need to omit `$kind` if `kind == 'rfq'`.

Reply:
absolutely correct, ty!